docs/documentation/concepts/index.md

```diff
@@ -27,8 +27,8 @@ Things do not have to be physical devices; they can also represent a web service
 Things expose their capabilities through **Channels**.
 Whether an installation takes advantage of a particular capability reflected by a Channel depends on whether it has been configured to do so.
 When you configure your system, you do not necessarily have to use every capability offered by a Thing.
-You can find out what Channels are available for a Thing by looking at the documentation of the Thing's **Binding**.
```

Comment:
Why drop this sentence? It may just refer to the new section "Bindings".

Reply:
Thank you - I agree and will put it back.